frontend/javascripts/messages.js

```diff
@@ -40,6 +40,7 @@ export const settings = {
   mergerMode: "Enable Merger Mode",
   gpuMemoryFactor: "Hardware Utilization",
   autoBrush: "Automatic Brush (Beta)",
+  overwriteMode: "Overwrite Mode (controls the overwrite behavior for volume annotation tools)",
```

Comment:
```suggestion
  overwriteMode: "Volume Annotation Overwrite Mode",
```

Sorry for the nitpicking, but this label is shown in tables (task type creation and if the user opens a task with recommended settings) and should be rather concise. The `loadingStrategyDescription` which is longer is a special case which is probably shown in a tooltip or so.

Reply:
Very good point! Thank you for being so attentive!